From c1b7d1d84500c22c1697b4112a58d8a5a5215c62 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Wed, 17 Mar 2021 15:14:28 +0700
Subject: [PATCH 124/135] arm64: dts: renesas: rzg2-smarc: Add comments about
 audio mixer settings

This patch adds comments about mixer settings which recommended when
playback/capture audio in RZ/G2L platform.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi b/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi
index f85588a..4133a95 100644
--- a/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzg2-smarc.dtsi
@@ -8,6 +8,21 @@
 #include <dt-bindings/pinctrl/pinctrl-rzg2l.h>
 #include <dt-bindings/gpio/gpio.h>
 
+/*
+ * SSI-WM8978
+ *
+ * This command is required when Playback/Capture
+ *
+ *	amixer cset name='Left Input Mixer L2 Switch' on
+ *	amixer cset name='Right Input Mixer R2 Switch' on
+ *	amixer cset name='Headphone Playback Volume' 100
+ *	amixer cset name='PCM Volume' 100%
+ *	amixer cset name='Input PGA Volume' 25
+ *
+ * ADV7535 Audio playback
+ *	aplay -D hw:1,0 xxx.wav
+ */
+
 / {
 	aliases {
 		serial0 = &scif0;
-- 
2.7.4

